quickstarts/calico/config.yml

```diff
@@ -0,0 +1,48 @@
+id:
+slug: calico newrelic
```

Comment:
User's know this is related to new relic since it will exist within the platform. Would you consider just `calico` as the slug? If you'd like to keep the full `calico newrelic` then we just need to hyphenate it, `calico-newrelic`

Reply:
Fixed